source/threads.tex

```diff
@@ -119,13 +119,13 @@
 time $C_n \geq C_t$, i.e., waking at $C_t + D_i + D_m$.
```

Comment:
The note 7 lines above is no longer a note in the wording paper, but there is no "removed" markup for this change. I've checked with the paper author, and this note was indeed intended to be made normative by this paper (as requested by LWG).